src/ReferenceWidget.vue

```diff
@@ -4,7 +4,11 @@
 			<div ref="customWidget" />
 		</div>
 
-		<a v-else-if="!noAccess && reference && reference.openGraphObject && !hasCustomWidget" :href="reference.openGraphObject.link" class="widget-default">
+		<a v-else-if="!noAccess && reference && reference.openGraphObject && !hasCustomWidget"
```

Comment:
Why only if reference.openGraphObject?

Reply:
I didn't change the condition?!
This is the linking I found, is there more?

Reply:
Oh sorry. Yeah well ok then. reference.openGraphObject is always there anyway.